flake.nix

```
@@ -130,6 +130,9 @@
                 docker-credential-helpers # for `docker-credential-osxkeychain` command
 
                 foundry-bin
+
+                # geth node
```

```suggestion
                # provides abigen
```
Since we aren't using the geth node otherwise.